src/main/kotlin/uk/co/craigbass/pratura/domain/Currency.kt

```diff
@@ -0,0 +1,17 @@
+package uk.co.craigbass.pratura.domain
+
+import org.joda.money.*
+import org.joda.money.format.MoneyFormatterBuilder
+import java.math.BigDecimal
+import java.util.*
+
+class Currency(val currency: String, val country: String, val language: String) {
```

Comment:
I want to protect my use cases from knowing about locales / joda as much as possible. Think having the gateway construct these makes sense.

It's probably a good shout to have the locale created at construct time instead of at format time.